### PR TITLE
Feature/userpage responsive

### PR DESCRIPTION
**Pull request by MishShevtsov**

### CODE REVIEW

**Comment by mplesha** [comment]
conflicts

**Comment by KimAT25** [comment]
Generally:
Please solve conflicts, add new line in the end of code, fix alignment where two tags in the line or >two attributes inside tag, delete all commented code.

**Comment by mplesha** [comment]
conflicted

File: src/app/component/user/components/profile/achievements-friends/achievements-friends.component.html [hunk 1]
```diff
@@ -2,10 +2,11 @@
...
       <div class="text-main-content">
-        <h3 class="text-title">
-          My Achievements
-        </h3>
-        <div class="text-number">8</div>
+        <div class="text-title">
+          My Achievements 
+          <span class="text-number">8</span>
+        </div>
+        <!-- <div class="text-number">8</div> -->
```
Comment (mplesha) [inline src/app/component/user/components/profile/achievements-friends/achievements-friends.component.html:9]: commented code

**Comment by mplesha** [comment]
There are still lines of commented code

File: src/app/component/user/components/profile/achievements-friends/achievements-friends.component.html (hunk 1 above)
Comment (MishShevtsov) [reply src/app/component/user/components/profile/achievements-friends/achievements-friends.component.html]: done

**Pull request merged**